### ISSUE TITLE
remove geo_vs_neff.npy

### ISSUE THREAD

**Issue by lukasc-ubc** [SiEPIC/SiEPIC-Tools#134]
@jaspreetj 

Regarding the file you added, it is huge, and takes forever for people to install.

https://github.com/SiEPIC/SiEPIC-Tools/blob/master/klayout_dot_config/python/SiEPIC/lumerical/geo_vs_neff.npy

Could you please:

1) compress it?  @mustafacc suggests npz format

2) this file should be PDK and Waveguide-type specific.  So it should go into each PDK, not in the Tool.  And there should be multiple of these files, one per waveguide.  I suggest supporting only strip waveguides in the EBeam PDK for now.  And adding in your function a check on a per active technology PDK basis that checks if it there.

thanks
Lukas

**Comment by jaspreetj** [comment]
I agree, we should just remove the file from tools altogether, and add it to a PDK instead. I need to check what use-cases are we using the file for and remove it accordingly while patching the use-cases.
